Formula/qwerty-fr.rb

```diff
@@ -0,0 +1,40 @@
+class QwertyFr < Formula
+  desc "Qwerty keyboard layout with French accents"
+  homepage "https://qwerty-fr.org"
+  url "https://github.com/qwerty-fr/qwerty-fr/releases/download/v0.7/qwerty-fr_0.7_mac.zip"
+  sha256 "34bc8c8f764798ed7e48a3f2404319e09a6d8bfcbef350b25df32387a3199bfa"
+  license "MIT"
+
+  def install
+    # Homebrew automatically extracts the folder inside the archive to the cur directory (not configurable)
+    # so we need to rebundle it
+    mkdir "qwerty-fr.bundle"
+    mv "Contents", "qwerty-fr.bundle/"
+    prefix.install "qwerty-fr.bundle"
+  end
+
+  def caveats
+    <<~EOS
+      1) Finishing the installation
+
+        To globally install the layout, you need to manually link the layout bundle (only once):
+          sudo ln -sf '#{prefix}/qwerty-fr.bundle' '/Library/Keyboard Layouts/'
+
+        If you only want to only install the layout for the current user, do the following instead:
+          ln -sf '#{prefix}/qwerty-fr.bundle' '~/Library/Keyboard Layouts/'
+
+
+      2) Restart your system (dead keys don't work properly before restarting).
+
+
+      3) You can then enable the qwerty-fr layout by doing the following:
+          a) Open "System Preferences" → "Keyboard" → "Input Source".
+          b) Click on "+" → Search for "qwerty-fr" → Click on "Add".
+          c) Click on the layout selection icon in the menu bar of macOS and choose "qwerty-fr"
+    EOS
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
There is nothing to test because it's ultimately just a config. What do you think?

Reply:
If we can't test it we can't ship distribute it.